### ISSUE TITLE
Pre-generate commands

### ISSUE THREAD

**Issue by fantoine** [ExternKit/extern-js-redis#1]
Maybe you could pre-generate the Haxe code for Redis commands. It will avoid to add compilation time to each compilation because of the macro.

I used this for the [AWS externs](https://github.com/ExternKit/extern-js-aws-sdk/tree/master/tools) which directly get the list of resources from an `npm update` of the aws sdk.
You could handle it in the same way with the redis-commands package.

**Comment by kevinresol** [comment]
Good idea. I will see if I have some time to get this done. (Or there are any volunteers? 😛 )
